extensions/OrtExt.mm

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import  <Foundation/Foundation.h>
+
+#include "onnxruntime/onnxruntime_cxx_api.h"
+#include "onnxruntime_extensions/onnxruntime_extensions.h"
+
+@interface OrtExt : NSObject
+
+// This is like a stub function where it calls `RegisterCustomOp` method from ORT Extensions
+// so that the function gets used directly otherwise registering custom ops using name would raise
+// a symbol not found error as it has been thrown away (as unused) during linking stage
++ (void)extensionsIncludedInPackage;
```

Comment:
User is not supposed to call this function. User will only do the call at swift level by registering custom ops using function name:
```
   // Register Custom Ops library using function name
   try options.registerCustomOps(usingFunction: "RegisterCustomOps")
```

Reply:
does the function ever get called?

Reply:
good question. This function is not explicitly called anywhere and I don't it's accessible to the user either.  but it does gets compiled and included in the built swift package/ spm extensions module and thus make the registercustomops symbols available.

Reply:
ok. the fact that this function is not actually called was not obvious to me. maybe including that in the comments would be helpful too. I wonder if it is possible for SPM to get "smarter" in the future and throw this unused function away too.

> For this to be equivalent with the cocoapods setup either the Swift user needs to be able to pull in the extensions header to use the RegisterCustomOps function pointer (not sure that is possible and it may be too late)

if the onnxruntime_extensions.h header doesn't get automatically imported into Swift or it is not easy to set that up, perhaps another option is to provide a public function that returns the address of `RegisterCustomOps` in OrtExt.h/mm.

Reply:
> I wonder if it is possible for SPM to get "smarter" in the future and throw this unused function away too.

understand the concern. FWIW, I found that when integrating Objective-C code into a Swift package, SPM generally follows the conventions of the Objective-C runtime and does not optimize away methods declared in Objective-C headers. so in other words, even if a method is declared in an objc header but is not explicitly called in the Swift code of the package, the Objective-C runtime mechanisms still recognize and maintain that method as part of the class's interface.

updated to add a note in the comment and added a OrtExt.h header for explicitly declaring this method.